shell/platform/android/io/flutter/embedding/android/FlutterFragment.java

```diff
@@ -476,9 +493,9 @@ public CachedEngineFragmentBuilder(
     }
 
     /**
-     * Pass {@code true} to destroy the cached {@link io.flutter.embedding.engine.FlutterEngine} when this {@code
-     * FlutterFragment} is destroyed, or {@code false} for the cached {@link io.flutter.embedding.engine.FlutterEngine} to
-     * outlive this {@code FlutterFragment}.
+     * Pass {@code true} to destroy the cached {@link io.flutter.embedding.engine.FlutterEngine}
```

Comment:
in some cases the existing comments use "true" and in others `{@code true}

Reply:
I didn't touch these pieces, the linter doesn't care.